tyson/README.md

```diff
@@ -1,7 +1,11 @@
 # TySON 🥊
-### TypeScript as a Configuration Language
+
+**Use TypeScript as a configuration language.**
```

Comment:
This is what I was seeing as revert:
1. We switched the language to `TypeScript as a Configuration Language`
2. I actually do prefer to use `###` here rather than `**` – it's what we had been doing in the Devbox repo as well. I both like the line that the header creates, and I think from an SEO perspective it's preferable for the HTML to end w/ a header tag.